src/sec_basic_diff_rules.mbx

```diff
@@ -121,8 +121,8 @@
           <li xml:id="constant-multiple-derivative-rule"><title>Constant Multiple Rule</title>
             <p>
             <md>
-              <mrow>\lzoo{x}{c\cdot f(x)} \amp= c\cdot\lzoo{x}{f(x)}</mrow> 
-              <mrow>\amp = c\cdot\fp(x)\text{.}</mrow> 
+              <mrow>\lzoo{x}{c\cdot f(x)} \amp= c\cdot\lzoo{x}{f(x)}</mrow>
+              <mrow>\amp = c\cdot\fp(x)\text{.}</mrow>
```

Comment:
This particular line shows up in the diff because I removed a trailing space character from the source. The `\text{.}` is still there from when it was automatically put there. 

In other places, I have found punctuation within an `md`, but not in any `mrow`. This causes a DTD violation, which I could theoretically ignore. The problem is that when I am trying to comb through DTD errors looking for ones that should be addressed, it's much easier if there is no clutter in the DTD error log. So I have been moving such punctuation. 

I'm afraid I have not been consistent with where I move it though. Inside the last `mrow` or after the `md`. If the current official position is that it should be inside the `md` but outside any `mrow`, then wherever I am moving it is not the right place. And yet as described above, I have a practical reason to clear the DTD violation.